### DISCUSSION TITLE
Files viewed incorrect count

### DISCUSSION THREAD

**Discussion by andretrevas** [community/community#16830] [Pull Requests]
We are working on a project and since last commit there are something strange. Someone open a pull request, I asked for modifications and he did. After this commit, all files were checked as viewed, the changes wasn't show and the count was incorrect (showing 15 of 20 files viewed, but all files were checked). I ignored that and we continue to work. We opened a new branch, new pull request and the error continues, as you can see in next image. 

![image](https://user-images.githubusercontent.com/8703140/168951833-751ea2b0-a55c-4d68-a15d-9d3a01a919b0.png)

This is the first time that I see this PR, and it has 17 of 23 files marked as viewed and 6 open to review. Some files has "Changed since last view" and if I unchecked any file, files viewed become negative

**Comment by LudBjork** [comment]
_I'll leave a comment here as my feedback relates to the same problem._

*Recreating the bug*
When I get a notifcation for reviewing, I go to the Notifications page to access the most recent changes. Primarily so I can review changed files only. 


I am experiencing a similar problem. For example if a PR contains 10 files, the "Files Viewed" counter thinks I've viewed all changed files of a PR, when 2 files remain not viewed.

At the present moment I am unable to re-create the issue, but will attach a screenshot next time I experience it.

**An example is shown below, where the counter is at max before reviewing:**
<img width="498" alt="bilde" src="https://github.com/community/community/assets/90049756/26d05aab-96f9-407a-94c3-d5e8777782ae">

Ticking viewed on and off subtracts the number of "Viewed".
